pkg/coderefs/coderefs.go

```diff
@@ -247,6 +247,12 @@ func (g grepResultLines) makeReferenceHunksReps(projKey string, ctxLines int) []
 
 		hunks := fileGrepResults.makeHunkReps(projKey, ctxLines)
 
+		if len(hunks) == 0 {
+			log.Error.Printf("unknown error scanning for code references in %s", fileGrepResults.path)
```

Comment:
I'd suggest saying something like `expected code references but found none in "%s"`